src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -613,6 +610,8 @@ public void Activate()
 
         public void Dispose()
         {
+            _inputPane?.Dispose();
+            _inputPane = null;
```

Comment:
We already dispose pane on WM_DESTROY. And it should be guaranteed to be called unless process was terminated (in which case we don't care about disposing).
https://github.com/AvaloniaUI/Avalonia/blob/41e86fb2a2a4137695dfe23625895c08e6274dc2/src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs#L112

Reply:
Other than that lgtm